setup.py

```diff
@@ -91,6 +91,7 @@ def run(self):
 
 install_requires = [
     'nvidia-ml-py>=12.535.108',  # see #107, #143, #161
+    'rocmi>=0.3', # see #137
```

Comment:
Can we install rocmi w/o AMD GPUs?

Reply:
Yes, it can be installed. It wil just report that there are no AMD gpus.